examples/ensemble/plot_gradient_boosting_regularization.py

```diff
@@ -38,11 +38,11 @@
 # map labels from {-1, 1} to {0, 1}
 labels, y = np.unique(y, return_inverse=True)
 
-X_train, X_test = X[:2000], X[2000:]
-y_train, y_test = y[:2000], y[2000:]
+X_train, X_test = X[:1500], X[1500:]
+y_train, y_test = y[:1500], y[1500:]
```

Comment:
you could also try reducing the number of samples in the `make_hastie_10_2` line above

Reply:
Have reduced both number of samples and number of estimators to get down to 5s.
Output below. Main message is still obvious I think.
![image](https://user-images.githubusercontent.com/43181038/141522214-7caa514e-a1e0-426a-911b-d90f7956824e.png)